lib/mochaReporter.js

```diff
@@ -49,18 +49,27 @@ class RPReporter extends Mocha.reporters.Base {
 
   onLaunchStart() {
     try {
+      const launchesAttributes = this.options.reporterOptions.attributes || [];
+      if (this.options.reporterOptions.skippedIssue === false) {
```

Comment:
This system attribute should be added only with `false` value?

Reply:
Yes, it's required in case of reporting skipped Issue without defect type (strictly false value). The same behavior as in java agents.